client/trino-client/src/main/java/io/trino/client/StatementStats.java

```diff
@@ -51,7 +51,8 @@
     public StatementStats(
             @JsonProperty("state") String state,
             @JsonProperty("queued") boolean queued,
-            @JsonProperty("scheduled") boolean scheduled,
+            @JsonProperty("progressPercentage") OptionalDouble progressPercentage,
```

Comment:
Have we considered `finishedPercentage` and `runningPercentage` and using them to draw progress bar for both, fte and non fte instead of `progressPercentage` + `faultTolerantExecutionRunningPercentage`?

Reply:
I think it makes sense, however currently Galaxy progress bar relies on `progressPercentage` [here](https://github.com/starburstdata/query-editor/blob/24b2bfe616995033ab44830912dbdf980571b2c2/src/components/results/stats/QueryStatus.tsx).

Reply:
I think it makes sense. `finishedPercentage` actually means `progressPercentage`.